tests/build-conf/qemux86-64-bios-grub-gpt/local.conf

```diff
@@ -235,6 +235,7 @@ BB_DISKMON_DIRS = "\
 # libsdl library available on your build system.
 #PACKAGECONFIG_append_pn-qemu-native = " sdl"
 #PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
+PACKAGECONFIG_append_pn-gnutls = " p11-kit"
```

Comment:
So all these are the same as the `PACKAGECONFIG_append = " p11-kit"` in `gnutls_%.bbappend `, right? I think it belongs to a recipe and we should not need to touch any of the `local.conf`.

Reply:
Correct, these are not needed as the `PACKAGECONFIG` change is global for the qemu layer.